common-service-core/src/main/java/org/zowe/apiml/security/SecurityUtils.java

```diff
@@ -37,7 +41,7 @@
      * @return {@link PrivateKey} or {@link javax.crypto.SecretKey} from keystore or key ring
      */
     public static Key loadKey(HttpsConfig config) {
-        if (config.getKeyStore() != null) {
+        if (StringUtils.isNotEmpty(config.getKeyStore())) {
```

Comment:
there are more parameters in key store configuration so it would be misleading, we cant find better solution now so we should leave it for the future